RibonDAO/manager
# (TECH-139) Cause details and Causes List

# Description

- Add page of list causes and causes details

# Necessary changes

- Route to list causes and details causes
- Tests and hooks to list causes and list a specific cause 

## Does this pull request close any open issues?

- Put here the issue that must be closed with this PR
- closes [#TECH-139](https://ribon.atlassian.net/jira/software/c/projects/TECH/boards/26?modal=detail&selectedIssue=TECH-139)

## Type of change

Please delete options that are not relevant.


## How to test

- Go to page of causes and see if all this causes are list and click one to see details
